File: dataset/output/PERPLEXITY_llama-3-sonar-large-32k-chat/Task99_PERPLEXITY_llama-3-sonar-large-32k-chat.cpp
#include <iostream>
#include <fstream>
#include <regex>

int main() {
    std::ifstream file("input.txt");
    std::string line;
    std::regex pattern("^\\d+.*\\.$");
    while (std::getline(file, line)) {
        if (std::regex_match(line, pattern)) {
            std::cout << line << std::endl;
        }
    }
    return 0;
}

File: dataset/output/PERPLEXITY_llama-3-sonar-large-32k-chat/Task99_PERPLEXITY_llama-3-sonar-large-32k-chat.java
import java.io.BufferedReader;
import java.io.FileReader;
import java.io.IOException;
import java.util.regex.Pattern;
import java.util.regex.Matcher;

public class Main {
    public static void main(String[] args) {
        try (BufferedReader br = new BufferedReader(new FileReader("input.txt"))) {
            String line;
            Pattern p = Pattern.compile("^[0-9]+.*\\.$");
            while ((line = br.readLine()) != null) {
                Matcher m = p.matcher(line);
                if (m.matches()) {
                    System.out.println(line);
                }
            }
        } catch (IOException e) {
            e.printStackTrace();
        }
    }
}